### ISSUE TITLE
Landscape Fixes for Grass Mods - Cutting Room Floor + Cyclic Interaction

### ISSUE THREAD

**Issue by ghost** [loot/skyrimse#225]
Landscape Fixes for Grass Mods author Code1k recommends it to be loaded after CRF.

[Photo of aforementioned recommendation for visibility, found in Landscape Fixes posts section.](https://imgur.com/a/tpTPmsR)

However, I encountered a cyclic interaction after manually editing the meta data.

Error: 
`Cyclic interaction detected between "Immersive Sounds - Compendium.esp" and "Cutting Room Floor.esp". Back cycle: Cutting Room Floor.esp, Landscape Fixes For Grass Mods.esp, Immersive Sounds - Compendium.esp`

[Landscape Fixes for Grass Mods](https://www.nexusmods.com/skyrimspecialedition/mods/9005?tab=posts)
[Cutting Room Floor](https://www.nexusmods.com/skyrimspecialedition/mods/276)
[Immersive Sounds - Compendium](https://www.nexusmods.com/skyrimspecialedition/mods/523)

**Comment by MacSplody** [comment]
@bpipher0225 Move Landscape Fixes for Grass Mods to the Large overhauls Group & the load after rule will work. 

The author is basing that off of cutting room floors old priority, which would have put it before almost everything. It's a few records and Loot sorts CRF after Landscape fixes if they're in the same group with no rules. I'll have to test it in game to see if it makes a difference.

**Comment by MacSplody** [comment]
@bpipher0225 I have tested it in game and yes it does make a difference. Landscape fixes for grass mods should be loaded after cutting room floor. The reason I had moved it back was from a note on the description page. 
> Load before any mods that change landscape height in the areas this mod changes.

A few mods I've noticed so far that also conflict with landscape fixes.
Moon & Star, Helgen Reborn, Undeath.

Thanks for informing me, I hadn't noticed that note in the download description.
